lib/pbench/test/unit/server/query_apis/test_datasets_publish.py

```diff
@@ -83,6 +83,11 @@ def _setup(self, client):
             payload={"controller": "node", "name": "drb", "access": "public"},
         )
 
+    def test_malformed_authorization_header(
+        self, client, server_config, attach_dataset
+    ):
+        super().test_malformed_authorization_header(client, server_config)
```

Comment:
Yeah I spent some time thinking if this is right flow of logic we have in our dataset publish where we query the dataset and then the user and then we check the authorization or should it be first authorization and then everything else. But then  to check the authorization we need a user which can only be queried from the dataset as the owner. So I kept it as it is for now and make the appropriate changes in the dataset publish test. And thats why I have to override the malformed authorization test in publish since we need a dataset before we check the authentication.